src/elements/button/button.svelte

```diff
@@ -121,6 +125,7 @@ $: {
 
 <style>
 :host {
-  display: inline-block !important;
+  --button-display: {display};
+  display: var(--button-display) !important;
```

Comment:
You can just prefix the classes above with `!` to avoid having to do this:
```
$: display = width === 'full' ? '!block' : '!inline-block';
```
why is important necessary though?